sessionctx/variable/tidb_vars.go

```diff
@@ -776,6 +776,8 @@ const (
 	TiDBMaxAutoAnalyzeTime = "tidb_max_auto_analyze_time"
 	// TiDBEnableConcurrentDDL indicates whether to enable the new DDL framework.
 	TiDBEnableConcurrentDDL = "tidb_enable_concurrent_ddl"
+	// TiDBEnableDistributeReorg indicates whether to enable the new Reorg framework.
```

Comment:
Maybe `TiDBDDLEnableDistributeReorg` is better? It is more unified with other DDL variables.